docs/faq.md

```diff
@@ -4,6 +4,10 @@ sidebar_position: 2
 
 # Frequently asked questions
 
+## What is conda?
+
+[Conda](https://docs.conda.io/en/latest/) is an OS-agnostic, [open source](https://github.com/conda/conda#readme) package manager and environment management system that helps you find and install packages as well as their dependencies. Though conda is written in Python and was created mainly for Python programs, it can package and distribute software for any language.
```

Comment:
It'd be great to define OS-agnostic in some way, though I'm unsure how without messing with the flow of your definition or linking out to a random site.

Reply:
But if we're gonna go that far, we'd probably need to define packages and environment management and stuff like that, too. I suppose we can assume a certain level of familiarity for now? Might be good for the FAQ to expand out on more novice topics like that. "Why do I need environment management?" "What does OS-agnostic mean?" "What are packages for?" Etc.

Reply:
It's not OS-agnostic, though. We could say "supports all major operating systems Microsoft Windows, Linux and Apple macOS"

Reply:
Completely agree! I just wanted to keep the part of my comments above about maybe added in "Why do I need environment management?" and "What are packages for?" to the FAQ list.